stdlib/public/Platform/winsdk.modulemap

```diff
@@ -153,6 +153,44 @@ module WinSDK [system] {
     }
   }
 
+  module DirectX {
+    module D3D12 {
+      header "d3d12.h"
+      export *
+
+      link "d3d12.lib"
+    }
+
+    module DXCore {
+      header "dxcore.h"
+      header "dxcore_interface.h"
+      export *
+
+      link "dxcore.lib"
```

Comment:
Just want to point out that this library requires a newer Windows SDK (for example, 10.0.19041.0) than the one currently mentioned on [swift.org](https://swift.org/download/#using-downloads) (10.0.17763.0).
`dxcore.lib` isn't present in 10.0.17763.0, and it causes a linker error.

Reply:
Ah, thanks, I'll remove the module definition.  It was stated above that it isn't necessary.  It is unfortunate that we cannot add definitions for future versions.